src/Zynga/Framework/Type/V1/NonBlankStringBoxTest.hh

```diff
@@ -34,6 +34,8 @@ class NonBlankStringBoxTest extends TestCase {
     $values[] = '';
     $values[] = ' ';
     $values[] = '    ';
+    $values[] = "\n";
+    $values[] = "\t";
 
     return $values;
```

Comment:
What about also handling '\r\n' for Mac line endings?

Reply:
Thanks - will add that test.
I agree that having some looser coupling between validators and StringBox would be a good improvement. Some kind of chain of validators would be useful, though I'm not sure I would use Decorator pattern, as seems like we're trying to modify existing function (import/instantiation) as opposed to adding new behavior. Would like to chat more and add that feature, but probably out of scope for current project.